packages/governance/src/types.js

```diff
@@ -0,0 +1,41 @@
+// @ts-check
+
+/**
+ * @typedef { 'amount' | 'unknown' | 'brand' | 'installation' | 'instance' | 'nat' | 'ratio' | 'string' } ParamType
+ */
+
+/**
+ * @typedef { Amount | Brand | Instance| Installation | bigint | Ratio | string | unknown } ParamValue
+ */
+
+/**
+ * @typedef  {Object} ParamDetails
```

Comment:
This type doesn't seem to be used at all. Seems like ParamDescription is the same thing and is the one used.